src/translate.zig

```diff
@@ -180,6 +180,20 @@ pub fn u32_from_object(env: c.napi_env, object: c.napi_value, comptime key: [:0]
     return u32_from_value(env, property, key);
 }
 
+pub fn u16_from_object(env: c.napi_env, object: c.napi_value, comptime key: [:0]const u8) !u16 {
+    var property: c.napi_value = undefined;
+    if (c.napi_get_named_property(env, object, key, &property) != .napi_ok) {
+        return throw(env, key ++ " must be defined");
+    }
+
+    const result = try u32_from_value(env, property, key);
```

Comment:
What do you think of replacing these lines with a call to `u32_from_object()` since the checks are the same, and then handle the error here?